proc_CPMG.py

```diff
@@ -79,24 +69,27 @@ def print_fun(x, f, accepted):
     fl.image(s.imag)
     #}}}
     #{{{select echo decay fit function
-    data = s['t2':(-200,200)].sum('t2')
+    s.ift('t2')
+    s = s['t2':(0,None)]
+    #s *= -1
+    s['t2',0] *= 0.5
```

Comment:
incorporated in FID slice function

Reply:
to be more specific:

1. why are you taking an FID slice inside the CPMG?
2. if you are taking an FID slice, why not use the FID slicing function?

Reply:
I misunderstood what you were asking.
1. the function of FID slice is slightly different than what is done here, you can go back to the original Slice_FID_from_echo function to see difference, this is I believe just slicing out the portion of the time domain that is applicable.
2. So my previous misunderstanding was that you wanted the function to be implemented in the proc_CPMG and so that is what is there in my latest push

Reply:
This has been changed I believe according to PR#14

Reply:
I diff against `alec_fitdata_rebased` and neither addresses my question:

> Why are we slicing out half of the echo?

More specifically (I said this in [a conversation on slack with Alec](https://jmfrancklab.slack.com/archives/CKRNAS3PD/p1592923328010100?thread_ts=1592922342.007900), so it might be good to go look at that and then touch base w/ him), why not (1) apply `hermitian_function_test` to find the center of the echo then (2) run `register_axis({'t2':0})` and (3) slice out the echo as large as you can, but still symmetrically about 0 (4) double-double check that the `t2` axis is symmetric about zero (`s.getaxis('t')[0]` is the same as `...[-1]`) (5) when you FT along `t2` the signal should now be entirely red.

Reply:
So I have incorporated the hermitian function test and register axis as you said. It looks the same (tE axis extends from 0 to about 2.8). Its also blue when ft'd but please take a look, something shifts it in the frequency domain but idk what....

Reply:
I have incorporated these in, its actually blue when ft'ed and shifted which is weird...

Reply:
I'm closing this for now, mentioned in an issue and not really part of this pull request.